static/app/components/sidebar/sidebarItem.tsx

```diff
@@ -65,6 +65,10 @@ type Props = ReactRouter.WithRouterProps & {
    * Sidebar is at "top" or "left" of screen
    */
   orientation: SidebarOrientation;
+  /**
+   * An optional prefix that can be used to reset the "new" indicator
+   */
+  isNewCacheBustKey?: string;
```

Comment:
Hmm now that I look closer, I wonder if it would actually make more sense to be `isNewCachekey` since it's actually a override.

Reply:
lol sorry

Reply:
@EvanPurkhiser How about `isNewCacheKeySuffix`?

Reply:
good for me

Reply:
or should it just be `isNewSeenKeySufix`?

Reply:
LOL yea actually

Reply:
wait, you mean `isNewSeenKeySuffix` lol